defaults/main.yml

```diff
@@ -106,6 +106,9 @@ concurrency: 1
 # - "[section1]"
 # - "setting2 = True"
 
+rally_cirros_img_url: "http://download.cirros-cloud.net/0.6.2/"
+rally_cirros_img_name: "cirros-0.6.2-x86_64-disk.img"
```

Comment:
What about 
rally_cirros_img_name: "{{ rally_cirros_img_url | basename }}"
or move the basename idea to the task so no need to have the extra variable?

Reply:
Yup, better, fixed